test/staging/set-is-subset-of-empty-index.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2023 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: https://github.com/tc39/proposal-set-methods/pull/102
+features: [set-methods]
+---*/
+
+let firstSet = new Set('a', 'b');
```

Comment:
this test isn't correct; `new Set` takes one iterable, not two items.

Reply:
You are right. `new Set` should take an iterable.

Reply:
Did the test run in V8 CI? Pass?

Reply:
Yes. It seems that it gets the first item 'a' and puts it in the set and ignores the rest.